pkg/helpers/sqlrole.go

```diff
@@ -0,0 +1,33 @@
+package helpers
+
+type SQLRoleAction string
+
+const (
+	RoleActionDelete = "delete"
+	RoleActionCreate = "create"
+)
+
+type SQLRoleAndAction struct {
+	Role   string
+	Action SQLRoleAction
+}
+
+func DiffCurrentAndExpectedSQLRoles(currentRoles map[string]struct{}, expectedRoles map[string]struct{}) []SQLRoleAndAction {
```

Comment:
I think this would be cleaner returning adds and deletes as two slices - all the callers need to split them out them anyway. If you're worried about getting them muddled up you could make them a struct with Added and Deleted fields.

Reply:
Will do the struct instead.